types/index.d.ts

```diff
@@ -3,6 +3,16 @@ import {
   FastifyReply,
   FastifyPluginCallback
 } from 'fastify'
+import { FastifyError } from '@fastify/error'
+
+declare interface FastifyBearerAuthErrors {
+  FST_BEARER_AUTH_INVALID_KEYS_OPTION_TYPE: FastifyError
```

Comment:
didnt we have some smart generic type logic for FastifyError

Reply:
...I have no idea.

Reply:
@fastify/typescript Any thoughts on this?